status/README.md

```diff
@@ -34,6 +33,29 @@ Ex:
 set -g @catppuccin_date_time_icon "null"
 ```
 
+### Notes for TPM users
+
+Make sure you load the catppuccin theme prior to setting the status-left and/or status-left options. This allows the catppuccin settings to be defined in the global space so they can then be set. After these settings have been configured make sure to run the TPM command to load the modules. This runs the plugin and sets them into the status line.
+
+```bash
+# ... load catppuccin theme
+run '~/.config/tmux/plugins/tmux/catppuccin.tmux' # or where this file is located on your machine
+
+# ... catppuccin settings
```

Comment:
The `...` at the beginning isn't really needed, see https://www.thepunctuationguide.com/ellipses.html.

`...` is great when it's continuing on from a previous line and thought, for example:

```
# Load the catppuccin theme...
run '~/.config/tmux/plugins/tmux/catppuccin.tmux'

# ... and then set status-left & status-right
set -g status-left
```

I do apologise if this is overly pedantic! I'm keen to get this in a good place to help others :)

Reply:
No worries! I thought that was a stylistic choice in the comments on the main README and was trying to keep it consistent. I'll make that adjustment.